### ISSUE TITLE
Are we even using scale in L2Norm?

### ISSUE THREAD

**Issue by isalirezag** [ShuangXieIrene/ssds.pytorch#24]
I don not understand how the scale is used in L2Norm function
I appreciate it if someone can explain it for me please

**Comment by foreverYoungGitHub** [comment]
This is proposed by the original ssd paper. This is used because of the different scale between different layers in the VGG. In the other networks, we prefer to use the regular batch norm rather than L2 norm.
